collectors/build/jenkins-audit/src/main/java/com/capitalone/dashboard/model/AuditResult.java

```diff
@@ -0,0 +1,34 @@
+package com.capitalone.dashboard.model;
+
+import com.capitalone.dashboard.response.DashboardReviewResponse;
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "audit_results")
+public class AuditResult extends BaseModel {
+
+    private ObjectId dashboardId;
+    private DashboardReviewResponse dashboardReviewResponse;
+
+    @SuppressWarnings("PMD")
+    private long timestamp;
```

Comment:
use camel case for the variables

Reply:
never mind, timestamp is a single word :)

Reply:
OK